near-sdk/src/lib.rs

```diff
@@ -277,6 +277,63 @@ extern crate quickcheck;
 /// }
 /// ```
 ///
+/// ## `#[callback_unwrap]` (annotates function parameters)
```

Comment:
The whole anchor/heading of this attribute should be blaced after `#[handle_result]` attribute's anchor